README.md

```diff
@@ -10,6 +10,8 @@ Navigation: [Website][1] | **Server repository** | [Client repository][2] | [Dex
 [![Build Status](https://api.travis-ci.com/Zarel/Pokemon-Showdown.svg)](https://travis-ci.com/Zarel/Pokemon-Showdown)
 [![Dependency Status](https://david-dm.org/zarel/Pokemon-Showdown.svg)](https://david-dm.org/zarel/Pokemon-Showdown)
 [![devDependency Status](https://david-dm.org/zarel/Pokemon-Showdown/dev-status.svg)](https://david-dm.org/zarel/Pokemon-Showdown?type=dev)
+[![Code Quality: Javascript](https://img.shields.io/lgtm/grade/javascript/g/Zarel/Pokemon-Showdown.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/Zarel/Pokemon-Showdown/context:javascript)
```

Comment:
```suggestion
```

Okay, I'll take the alert count, but the code quality badge seems redundant/vanity.